File: tests/transpiler/x/cs/map_assign.cs
// Mochi 0.10.31 - generated 2025-07-19 12:59:48 UTC
using System;
using System.Collections.Generic;

class Program {
    static void Main() {
        var scores = new Dictionary<object, object>{{"alice", 1}};
        scores["bob"] = 2;
        Console.WriteLine(scores["bob"]);
    }
}

File: tests/transpiler/x/fs/map_assign.fs
// Generated 2025-07-21 15:37 +0700
open System

type Anon1 = {
    mutable alice: int
}
let mutable scores: Anon1 = { alice = 1 }
scores.["bob"] <- 2
printfn "%s" (string (scores.["bob"]))
